src/test/java/ua/ithillel/lesson23/HeroServiceIntegrationTest.java

```diff
@@ -0,0 +1,36 @@
+package ua.ithillel.lesson23;
+
+import ua.ithillel.lesson22.Hero;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HeroServiceIntegrationTest {
+
+    private HeroService target;
+    List<Hero> heroList = new ArrayList<>();
+
```

Comment:
```suggestion
@BeforeEach
```
зазвичай так анотується, і потім метод init не потрібно викликати напряму

Reply:
Точно. Забывается )